src/open_inwoner/accounts/views/invite.py

```diff
@@ -41,3 +43,30 @@ def get_object(self, queryset=None):
             raise Http404(_("The invitation was already used"))
 
         return invite
+
+    def dispatch(self, request, *args, **kwargs):
+        invite = self.get_object()
+        user = request.user
+
+        # mark invitation as accepted and add inviter to the user's contacts as well
+        if user.is_authenticated:
```

Comment:
Please only do this if the invite has not yet been accepted earlier.

Reply:
Done!
I hadn't added this here since we have an early check in `get_object()` where we show a 404 if the invite is already accepted.